app/models/cyclist.rb

```diff
@@ -0,0 +1,19 @@
+class Cyclist < SimpleDelegator
+  extend ActiveModel::Naming
+
+  def self.model_name
+    ActiveModel::Name.new(Cyclist)
+  end
+
+  def initialize(user)
+    user.cyclist = true
+    user.organic_growth_asserted = true
+    user.password = SecureRandom.base64
+    user.terms_and_conditions_accepted = true
```

Comment:
These overrides are:

* required fields and checkboxes that would have been filled out by prospective users
* a randomly generated `password`
* the `cyclist=true` flag, which kicks off the desired behavior

We wouldn't want to source these values from the outside world.